core/src/main/java/tachyon/client/ReadType.java

```diff
@@ -26,7 +26,12 @@
   /**
    * Read the file and cache it.
    */
-  CACHE(2);
+  CACHE(2),
+
+  /**
+   * Read the file and promote it back to top StorageTier
+   */
+  CACHE_PROMOTE(3);
```

Comment:
difference between CACHE and CACHE_PROMOTE is  local read, the remote read are the same.
cache_promote: if local tachyon space contains the bloock and the block is not on top storage layer, then promote the block to top storage layer.